packages/rx_bloc_cli/lib/src/processors/ios/info_plist_processor.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:rx_bloc_cli/src/extensions/string_buffer_extensions.dart';
+
+import '../common/abstract_processors.dart';
+
+/// String processor used for processing an Info.plist file in the ios
+/// directory found at: ios/Runner/Info.plist
+class InfoPlistProcessor extends StringProcessor {
+  /// Podfile file processor constructor
+  InfoPlistProcessor(super.args);
+
+  @override
+  String execute() {
+    if (input == null) return '';
+    final buffer = StringBuffer(input!);
+
+    //_updateGlobalIOSPlatform(buffer);
+    if (args.qrScannerEnabled) {
+      _updateInfoPlistPermissions(buffer);
+    }
+
+    return buffer.toString();
+  }
+
+  void _updateInfoPlistPermissions(StringBuffer buffer) {
+    const permissions = '''
+    <key>NSCameraUsageDescription</key>
+    <string>Your message to user when the camera is accessed for the first time</string>
```

Comment:
Even though we are generating a showcase, I am kind of divided on whether or not should we add a generic text (to be replaced) or a placeholder just for a showcase. Do you think it would be better if we provided a real message instead or keep the placeholder?

Reply:
Changed